.github/workflows/pr.yml

```diff
@@ -0,0 +1,22 @@
+name: Build
+
+on:
+  pull_request:
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: |
+          yarn install --frozen-lockfile
+
+      - name: Package extension
+        run: |
+          yarn package
```

Comment:
use `yarn compile` instead?

Reply:
It's important to ensure that we can package the extension as well, don't you think? 👀

Fine with removing this, but I feel this adds an easy way to catch some issues with packaging and it shouldn't affect checking transpiling in any way.